vertexai/genai/example_test.go

```diff
@@ -51,6 +51,31 @@ func ExampleGenerativeModel_GenerateContent() {
 	printResponse(resp)
 }
 
+// This example shows how to a configure a model. See [GenerationConfig]
+// for the complete set of configuration options.
+func ExampleGenerativeModel_GenerateContent_config() {
+	ctx := context.Background()
+	client, err := genai.NewClient(ctx, option.WithAPIKey(os.Getenv("GEMINI_API_KEY")))
```

Comment:
Shouldn't this be with project/location and not key?

Reply:
Yes, fixed.